egs/wsj/s5/utils/parse_options.sh

```diff
@@ -46,11 +46,12 @@ done
 ###
 while true; do
   [ -z "${1:-}" ] && break;  # break if there are no arguments
+  # shellcheck disable=2154
   case "$1" in
     # If the enclosing script is called with --help option, print the help 
     # message and exit.  Scripts should put help messages in $help_message
   --help|-h) if [ -z "$help_message" ]; then echo "No help found." 1>&2;
-	  else printf "$help_message\n" 1>&2 ; fi; 
+	  else printf "%s\n" "$help_message" 1>&2 ; fi; 
```

Comment:
Actully you can remove this whole help-message mechanism.  It seems to be only used by one script, local/score_combine.sh.   Just remove the variable from there, and remove this code.